rock_paper_scissors/rps.py

```diff
@@ -2,8 +2,37 @@
 
 import sys
 
+possible_plays = [['rock'], ['paper'], ['scissors']]
+
+# 2= [['rock', 'rock'], ['rock', 'paper'], ['rock', 'scissors'],
+    # ['paper', 'rock'], ['paper', 'paper'], ['paper', 'scissors'],
+    # ['scissors', 'rock'], ['scissors', 'paper'], ['scissors', 'scissors']]
+
 def rock_paper_scissors(n):
-  pass 
+    if n <= 0:
+        return [[]]
+    if n == 1:
+        return([['rock'], ['paper'], ['scissors']])
+    i = 0
+    j = 0
+    x = []
+    while i < 3:
+        rock = (possible_plays[0] + possible_plays[i] * (n - 1))
+        x.append(rock)
+        paper = (possible_plays[1] + possible_plays[i] * (n - 1))
+        x.append(paper)
+        scissors = (possible_plays[2] + possible_plays[i] * (n - 1))
+        x.append(scissors)
+        i += 1
```

Comment:
It looks like you were attempting an iterative solution to this so I am going to share with you how the solution utilized a stack to solve this iteratively. 
<img width="402" alt="Screen Shot 2019-06-10 at 10 47 39 AM" src="https://user-images.githubusercontent.com/33299372/59211557-5bd6a300-8b6d-11e9-9863-d529d9e54ecb.png">


However, I think that the recursive solution makes more sense to me so I want to share that with you as well.
<img width="454" alt="Screen Shot 2019-06-10 at 10 47 51 AM" src="https://user-images.githubusercontent.com/33299372/59211589-6e50dc80-8b6d-11e9-8377-ad6095695503.png">